lib/src/chain.dart

```diff
@@ -228,7 +229,8 @@ class Chain implements StackTrace {
   ///
   /// The trace version of a chain is just the concatenation of all the traces
   /// in the chain.
-  Trace toTrace() => new Trace(traces.expand((trace) => trace.frames));
+  Trace toTrace() => new Trace(traces.expand((trace) => trace.frames),
+      original: traces.map((trace) => trace.original).join('\n$chainGap'));
```

Comment:
One last thing: I don't think we should set an original here. This wasn't generated from a single original trace, and it's not going to be in a valid format anyway.